controllers/fenceagentsremediation_controller_test.go

```diff
@@ -200,11 +201,149 @@ var _ = Describe("FAR Controller", func() {
 				By("Still having one test pod")
 				verifyPodExists(testPodName)
 
-				By("Verifying correct conditions for unsuccessfull remediation")
+				By("Verifying correct conditions for unsuccessful remediation")
 				Expect(underTestFAR.Status.LastUpdateTime).ToNot(BeNil())
-				verifyStatusCondition(dummyNode, commonConditions.ProcessingType, conditionStatusPointer(metav1.ConditionFalse))
-				verifyStatusCondition(dummyNode, v1alpha1.FenceAgentActionSucceededType, conditionStatusPointer(metav1.ConditionFalse))
-				verifyStatusCondition(dummyNode, commonConditions.SucceededType, conditionStatusPointer(metav1.ConditionFalse))
+				verifyStatusCondition(underTestFAR.Status.Conditions, dummyNode, commonConditions.ProcessingType, conditionStatusPointer(metav1.ConditionFalse))
+				verifyStatusCondition(underTestFAR.Status.Conditions, dummyNode, utils.FenceAgentActionSucceededType, conditionStatusPointer(metav1.ConditionFalse))
+				verifyStatusCondition(underTestFAR.Status.Conditions, dummyNode, commonConditions.SucceededType, conditionStatusPointer(metav1.ConditionFalse))
+			})
+		})
+
+		Context("Fence Agent Failures", func() {
```

Comment:
Nit: no need for capital letters.
```suggestion
		Context("Fence agent failures", func() {
```